test/Recipe-test.js

```diff
@@ -23,35 +23,35 @@ describe('Recipe', () => {
     expect(recipe1.id).to.equal(595736)
```

Comment:
This comment is for line 13 (for some reason github won't let me add it there). 
I know we keep dancing back and forth on using sampleRecipeData[0] or the full hardcoded data. I see a benefit now if we included the full typed out, it would allow us to see the full data and make accurate comparisons to its expect statements instead of us having to refer to the sampleReciepeData file itself. I know it've been changed before but I would recommend, putting the entire recipe here so we can see what we're comparing it to.

Reply:
I would argue to keep it as it is, I think that if we do make that change we wouldn't be using the sample data at all in this test. Heather didn't have feedback on this and said "Good job creating a sample data file for your testing!"